fhir-parser-resources/mappings.py

```diff
@@ -80,17 +82,30 @@
 # For enum codes where a computer just cannot generate reasonable names
 enum_map = {
     '=': 'eq',
+	'!=': 'neq',
     '<': 'lt',
     '<=': 'lte',
     '>': 'gt',
     '>=': 'gte',
     '*': 'max',
+	'public': 'public_fhir',
+	'private': 'private_fhir'
```

Comment:
It would seem more parallel to add these to the `reservedmap` rather than here in the enum_map. That would follow better with the other reserved words in the `reservedmap` even those used as enum codes. Also, we're not consistent between quoting reserved words (such as `as`) vs adding an _fhir suffix as is done here.

Reply:
Moving public and private into reservedmap. Unsure on original design rules for choosing quoted vs _fhir suffix, so keeping suffix here.